app.py

```diff
@@ -39,13 +39,23 @@ def main() -> None:
         deploy_env=ENV,
     )
 
+    staging = StagingStack(
+        app,
+        "staging",
+        stack_name=f"{ENV}-geospatial-data-lake-staging",
+        env={"region": region, "account": account},
+        deploy_env=ENV,
+    )
+
     processing = ProcessingStack(
         app,
         "processing",
         stack_name=f"{ENV}-geospatial-data-lake-processing",
         env={"region": region, "account": account},
         deploy_env=ENV,
     )
+    processing.add_dependency(storage)
+    processing.add_dependency(staging)
```

Comment:
Why do we have to wait for these? Is it possible to rearrange things so that they can continue to be deployed in parallel?